spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -10,10 +18,34 @@
     end
 
     context "when sending a welcome_thread notification" do
-      xit "generates the appropriate broadcast to be sent to a user"
-      xit "it sends a welcome notification for that broadcast"
-      xit "it does not send duplicate welcome notification for that broadcast"
-      xit "does not send a notification to a user who has commented in a welcome thread"
+      let(:welcome_thread_article) { create(:article, title: "Welcome Thread - v3000", published: true) }
+      let!(:welcome_thread_comment) { create(:comment, commentable_id: welcome_thread_article.id, commentable_type: "Article", user_id: user.id) }
```

Comment:
```suggestion
      let!(:welcome_thread_comment) { create(:comment, commentable: welcome_thread_article, commentable_type: "Article", user: user) }
```
You already have the objects here, and you can create a comment by passing in `commentable` and `user`. You don't actually need to get the `id` off of them in order to create the comment.